src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private double base;
+    private double height;
+    private double hypotenuse;

```

Comment:
The variable 'hypotenuse' is not used anywhere else besides the constructor to calculate its value. It's not necessary as a class-level variable. Consider calculating it only when needed, for example in a method.